src/components/CDPDisplay/Presentation.js

```diff
@@ -56,6 +58,24 @@ export default function({ cdp, showSidebar, account, network }) {
   const isOwner = account && account.cdps.some(userCdp => userCdp.id === cdpId);
 
   const [actionShown, setActionShown] = useState(null);
+  const { show: showBanner } = useBanner();
+
+  const unlockedCollateral = getUnlockedCollateralAmount(cdp, false);
+  // TODO: this causes an infinite loop if uncommented:
+  // useMemo(() => {
+  //   console.log('cdp', cdp);
+  //   console.log('unlockedCollateral', unlockedCollateral);
+  //   if (unlockedCollateral > 0) {
+  //     showBanner({
+  //       banner: 'claim',
+  //       props: {
+  //         colName: 'testName',
+  //         amount: cdp.unlockedCollateral.toNumber(),
+  //         symbol: 'SYM'
+  //       }
+  //     });
+  //   }
+  // }, [cdp, showBanner, unlockedCollateral]);
```

Comment:
I thought adding the dependencies here would prevent the re-render issues, when `showBanner` is called it changes the DOM and triggers a render, which calls `showBanner` again, and so on.